crates/toolchain/src/pms/pnpm.rs

```diff
@@ -218,4 +218,30 @@ impl PackageManager<NodeTool> for PnpmTool {
 
         Ok(())
     }
+
+    async fn install_focused_dependencies(
+        &self,
+        toolchain: &Toolchain,
+        package_names: &[String],
+        production_only: bool,
+    ) -> Result<(), ToolchainError> {
+        let mut cmd = self.create_command();
```

Comment:
Does this need something like `cmd.arg("install");`? I might be missing where it adds the install command, but the other two package managers add it directly in their implementations.

Reply:
Yeah nice catch.